tests/data_context/store/test_datasource_store_cloud_backend.py

```diff
@@ -43,23 +45,30 @@ def test_datasource_store_create(
         "requests.get", autospec=True, side_effect=mocked_datasource_get_response
     ):
 
-        datasource_store_ge_cloud_backend.set(key=key, value=datasource_config)
+        saved_datasource_config: DatasourceConfig = (
```

Comment:
In `test_cloud_data_context_save_datasource_empty_store()` we were mainly testing that the `DatasourceStore.set()` method would correctly return the config. It makes more sense to test that functionality here with the datasource store tests.